.storybook/addons.js

```diff
@@ -1,3 +1,3 @@
-import '@storybook/addon-actions/register';
-import '@storybook/addon-links/register';
-import '@storybook/addon-knobs/register';
+import '@storybook/addon-actions/register';
+import '@storybook/addon-links/register';
+import '@storybook/addon-knobs/register';
```

Comment:
Looks like your editor might be removing the newline at the end of the files, can you check the configuration?
The linter will complain about this.

Reply:
Yes, I changed the configuration, it should be fixed.